App.js

```diff
@@ -56,13 +89,16 @@ export default function App() {
   });
   return (
     <BiteShareContext.Provider value={{ state, dispatch }}>
-      <ThemeProvider theme={theme}>
-        <NavigationContainer>
-          <HomeScreen />
-        </NavigationContainer>
-        {/* <DummyComponent /> */}
-      </ThemeProvider>
+      { !fontsLoaded
+        ? <AppLoading />
+        : (<ThemeProvider theme={theme}>
+          <NavigationContainer>
+            <HomeScreen />
+          </NavigationContainer>
+          {/* <DummyComponent /> */}
+        </ThemeProvider>)
+      }
```

Comment:
Hi, I did not understand the conditional here. If font is not loaded, then you're returning `<AppLoading/>` else you are returning the actual app code, is it so? Why are you returning the `<ThemeProvider />` wrapped component if the font is not loaded?

Reply:
I think the wrapped component is rendered if font is loaded?

Reply:
Got it. Thanks.

Reply:
yes. If load the fonts first, then load the <ThemeProvider />